### ISSUE TITLE
angular-fire + firebase functions breaks ng deploy

### ISSUE THREAD

**Issue by wagnermaciel** [angular/angularfire#2246]
### Bug description

I experienced a bug where `ng add @angular-fire` followed by `firebase init functions` broke `ng deploy` due to a bug in firebase-tools.

### Version info

**Angular CLI:**
8.3.19

**Firebase:**
\>= 5.5.7 <8

**AngularFire:**
5.2.3

### How to reproduce these conditions

`ng new`
`ng add @angular-fire`
`firebase init functions`
`ng deploy` (should break)

### Debug output

** Output from `ng deploy` **

![image](https://user-images.githubusercontent.com/25158423/68906883-a818a700-06fa-11ea-9014-2f15a7e50234.png)

** After reinstalling firebase-tools with `npm install --save-dev firebase-tools@latest` **

![image](https://user-images.githubusercontent.com/25158423/68906924-c9799300-06fa-11ea-8f77-8db381d8e9b7.png)

### Expected behavior

ng deploy deploys the application to Firebase.

### Actual behavior

You get an error
```TypeError: entry.message.split is not a function```

The error is coming from firebase-tools. The bug has been fixed in v7.7.0
[Link to the fix](https://github.com/firebase/firebase-tools/commit/f56b29ca5d0ec1157f77ed21713bdc204a5629ce)

TLDR; If you're getting this error, just run `npm install --save-dev firebase-tools@latest`

**Comment by wagnermaciel** [comment]
Also, you can observe the actual error message if you console log `entry` in `node_modules/@angular-devkit/core/src/logger/logger.js` `Logger.next`.

![image](https://user-images.githubusercontent.com/25158423/68908702-ed3fd780-0700-11ea-9608-c2bf347b90d3.png)

**Comment by SpiegelSoft** [comment]
I've got firebase-tools 7.12.1, and this is still broken.

**Comment by abraham** [comment]
This just started happening for me too.

**Comment by marcucio** [comment]
broken here too

**Comment by ammaarpatel99** [comment]
Broken for me too with firebase-tools at 7.16.1

**Comment by ammaarpatel99** [comment]
As a temporary fix it seems if you delete `firebase.json` and `.firebaserc` and then run `ng add @angular/fire` then it works again.
